### DISCUSSION TITLE
Add random word to website based aliases

### DISCUSSION THREAD

**Discussion by SourBadger** [simple-login/app#902] [Feature Requests]
When using the browser extension and clicking the SL icon next to email field, SL will generate the alias _website[at]mycustomdomain.com_. I instead want it to generate an alias like _website-randomword[at]mycustomdomain.com_. The advantage of including the website name is it make the alias more identifiable for me. However, it also makes it predictable for an attacker. An attacker can guess my email address when trying to compromise my account on a site.

**Comment by nguyenkims** [comment]
@SourBadger if you enable this option on your domain, a random suffix will be added to the generated alias.

<img width="558" alt="image" src="https://user-images.githubusercontent.com/3167780/164405516-07b45d80-51f7-4ed7-a36a-03585359a257.png">

**Reply by SourBadger** [reply]
Thank you so much! I was looking for this under the generic Settings page and didn't think to look under the Domains details. This does indeed address my ask.

**Comment by Heziode** [comment]
@nguyenkims How to enable this feature for the main domain, since we cannot see it in **Domains** section?

**Reply by nguyenkims** [reply]
What do you mean by "main" domain?

**Reply by Heziode** [reply]
~~Le domaine de utilisé par défaut, celui qui nous permet d'accéder à l'app (app.maindomain.com).~~

I mean the `mydomain.com` (referenced into the README).

**Reply by nguyenkims** [reply]
Oh I see, you can the main domain as a custom domain and enable the setting there. You can also manually mark this domain as verified by going to the `custom_domain` table in the database.

**Reply by Heziode** [reply]
@nguyenkims when I attempt to add the domain, I got the following error: `A custom demain cannot be a built-in domain`

**Reply by nguyenkims** [reply]
Oh I see, in this case you can maybe add the domain directly to the database then. In the next version, we'll try to add a way to have the main domain always added.

**Reply by Heziode** [reply]
Ok,

To do that, I found an answer here:

```sql
-- Get your user ID
SELECT id,email FROM users;

-- Add your domain
INSERT INTO custom_domain 
    (created_at,user_id,domain,verified) 
VALUES (NOW(),#YOUR_USER_ID#,'#YOUR.DOMAIN#',FALSE);
```

Source: https://github.com/simple-login/app/issues/382#issuecomment-929994588
